.github/workflows/project-codeflare-release.yml

```diff
@@ -184,3 +182,27 @@ jobs:
 
     - name: Verify generated file
       run: cat config/component_metadata.yaml
+
+  Update_release_version_info_to_confluence:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v4
+
+      - name: Trigger and Update Component Release Matrix Versions Info to Confluence
+        run: |
+            gh workflow run update_confluence.yml --ref updateReleaseDetails \
```

Comment:
This comment is just for tracking. This line should point to `update-release-matrix-to-confluence.yml` before merging.

Reply:
Sure , updated for testing purpose. will change it

Reply:
Done